src/BlazorUI/Bit.BlazorUI/Components/Lists/Timeline/BitTimeline.scss

```
@@ -13,46 +76,52 @@
-    --bit-tln-pcn-flexdir: row;
-    --bit-tln-scn-flexdir: row-reverse;
-}
-
 .bit-tln-itm {
     width: 100%;
     display: flex;
     min-height: 100%;
     position: relative;
     align-items: center;
     flex-direction: var(--bit-tln-itm-flexdir);
+    // Resolved per item so that an item-level color or size class also drives the padding of the
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)